### ISSUE TITLE
UserWarning when initializing PandasDataSet with pandas >= 0.21.0, ge == 0.3.2

### ISSUE THREAD

**Issue by alchenist** [fivetran/great_expectations#241]
Getting these UserWarnings about setting attributes on a DataFrame when using ge.read_csv(). Looks like this is associated with [pandas >= 0.21.0](https://github.com/pandas-dev/pandas/pull/16951)?

```
~\envs\conference_analysis\lib\site-packages\great_expectations\dataset\base.py:222: UserWarning: Pandas doesn't allow columns to be created via a new attribute name - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute-access
  "expectations" : []
~\envs\conference_analysis\lib\site-packages\great_expectations\dataset\base.py:228: UserWarning: Pandas doesn't allow columns to be created via a new attribute name - see https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute-access
  "output_format" : 'BASIC',
```

**Comment by abegong** [comment]
Good feedback!

This will be addressed in the upcoming v0.4 release.

If you use the `develop` branch, I believe the change is already implemented. See this commit: https://github.com/great-expectations/great_expectations/commit/4c8bfd9f0719fc7ace6a0efeb6e66200b74f4c8d

Heads up, though---there are a bunch of API changes in v0.4, so you may run into some compatibility issues with expectations from earlier versions.

**Comment by alchenist** [comment]
Thanks! Made the issue title more specific.
